news/2024/2024-08-28-osumania-4k-world-cup-2024-round-of-32.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "osu!mania 4K World Cup 2024: Round of 32 Recap"
+date: 2024-08-28 12:00:00 +0000
+---
+
+The key smashing has officially commenced last weekend, and now is the perfect time to pull out the Round of 32 highlight reel.
+
+![](/wiki/Tournaments/MWC/2024_4K/img/mwc4k2024-banner.jpg)
+
+We all know qualifiers have never been the deciding factor in a tournament. While they definitely provide an initial edge to those teams who prove to go above and beyond the competition, qualifiers are but the start of the show!
+
+This year we've witnessed everything, with this year's qualifiers being one of the most competitive to date. From the usual teams getting podium placements, to fan favourites such as Japan losing some of its previous years' lustre, or even seeing new faces in higher seeds such as the United Kingdom!
```

Comment:
the second sentence isn't complete

```suggestion
This year's qualifiers were one of the most competitive to date. From the usual teams getting podium placements, to fan favourites such as Japan losing some of their previous years' lustre, or even seeing new faces in higher seeds such as the United Kingdom, it seems this year we've witnessed just about everything!
```

Reply:
done in 9080ed36933fd600bbfb58bd84ff920ed2b6ebae